python/django_bolt/middleware/compiler.py

```
@@ -425,6 +432,39 @@ def add_optimization_flags_to_metadata(metadata: dict[str, Any] | None, handler_
+
+    Returns:
+        Normalized CIDR strings, for example `["10.0.0.0/8", "127.0.0.1/32"]`.
+
+    Raises:
+        ImproperlyConfigured: An entry is not an address or a CIDR block.
+    """
+    configured = getattr(settings, "BOLT_TRUSTED_PROXIES", None) or []
+    if isinstance(configured, str):
+        raise ImproperlyConfigured(
+            f"BOLT_TRUSTED_PROXIES must be a list of addresses or CIDR blocks, got the string {configured!r}."
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)